devicon.json

```
@@ -5868,6 +5868,34 @@
         "color": "#8dc63f",
         "aliases": []
     },
+    {
+        "name": "mithril",
+        "altnames": [
+            "mithril.js"
+    ],
+        "tags": [
+            "javascript",
+            "framework",
+            "frontend",
+            "js"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "line"
+            ],
+            "font": [
+                "original"
```

Line icon can also be added as a font :)
```suggestion
                "original",
                "line"
```